KEMP

```diff
@@ -0,0 +1,8 @@
+  #name of the company & the uri to apply 
+- perk: {company: "KEMP Tehcnologies", uri: "http://kemptechnologies.com/"}
+  #product/service description
+ d escription: "As a Microsoft MVP , you are entitled to a complimentary NFR copy of LoadMaster VLM-2000  load balancer is a device that acts as a reverse proxy and distributes network or application traffic across a number of servers. Load balancers are used to increase capacity (concurrent users) and reliability of applications."
```

Comment:
Can you please fix whitespacing on this line?

`d escription` --> `description`
`MVP , you` --> `MVP, you`

The description sentence doesn't sound right (note that there should not be any markup): 
`As a Microsoft MVP, you are entitled to a complimentary NFR copy of LoadMaster VLM-2000 is a device that ...`